AlembicImporter/Packages/com.unity.formats.alembic/Documentation~/com.unity.formats.alembic.md

```diff
@@ -103,7 +107,7 @@ The Timeline can be used to playback and record Alembic animation including:
 * Create sequences using Alembic clips with the ability to trim times and adjust clip-ins
 * Record Alembic data directly from the Timeline to an Alembic file
 
-
+> Consider making sure that you can actually edit Alembic Shot Clips via code or otherwise add it to known limitations
```

Comment:
where does it say  you can edit Alembic Shot Clips via code?

Reply:
ah ok. We already mention that there is no external API though. Would that not cover this as well?